templates/vue/src/components/Lightbox/media/MultiContentMedia/MultiContentRows.vue

```diff
@@ -1,6 +1,9 @@
 <template>
   <div data-qa="multi-content-rows" class="multi-content-rows">
-    <b-row :class="(node.presentationStyle === 'page' ? 'mr' : 'ml') + '-0'">
+    <div v-if="allHidden">
+      Content in this node has not been unlocked yet.

```

Comment:
Same here - let's please tell the user what they need to do to unlock this node.